components/controller.ts

```diff
@@ -869,19 +869,20 @@ export class Controller {
     }
 
     /**
-     * Loads all normal, pre-built or pure JS plugins.
+     * Loads all normal, pre-built or pure JS plugins form plugins folder.
      *
      * @internal
      */
     private async _loadPlugins(): Promise<void> {
-        const entries = (await readdir(process.cwd())).filter(
+        const entries = (await readdir(join(process.cwd(), "plugins"))).filter(
             (n) => isPlugin(n, "js") || isPlugin(n, "cjs"),
         )
 
         for (const plugin of entries) {
-            const src = (await readFile(plugin)).toString()
+            const sourceFile = join(process.cwd(), "plugins", plugin),
+            src = (await readFile(sourceFile)).toString();
```

Comment:
Style: please use 2 const declarations instead of comma-trailing them.  Also, run `yarn prettier`.